.eslintrc.js

```diff
@@ -1,78 +1,38 @@
 // @ts-check
-
```

Comment:
I simplified eslintrc a bit by removing the unused prettier plugin and moving all rules to the root object